sphinx/writers/html.py

```diff
@@ -874,11 +874,29 @@ def visit_field_name(self, node):
 
     def visit_math(self, node, math_env=''):
         # type: (nodes.Node, unicode) -> None
-        logger.warning(__('using "math" markup without a Sphinx math extension '
-                          'active, please use one of the math extensions '
-                          'described at http://sphinx-doc.org/en/master/ext/math.html'),
-                       location=(self.builder.current_docname, node.line))
-        raise nodes.SkipNode
+        name = self.builder.math_renderer
```

Comment:
I feel this code will return renderer object. How about `math_renderer_name`?

or how about:
```
visit, depart = self.builder.math_renderer
```

Reply:
done